cms/static/js/certificates/collections/certificates.js

```diff
@@ -0,0 +1,37 @@
+// Backbone.js Application Collection: Certificates
+
+define([
+    'backbone', 'js/certificates/models/certificate'
+],
+function(Backbone, Certificate) {
+    'use strict';
+    var CertificateCollection = Backbone.Collection.extend({
+        model: Certificate,
+
+        initialize: function(attr, options) {
+            // Set up the attributes for this collection instance
+            this.url = options.certificateUrl;
+        },
+
+        certificate_array: function(certificate_info) {
+            var return_array;
+            try {
+                return_array = JSON.parse(certificate_info);
+            } catch (e) {
+                return_array = certificate_info;
```

Comment:
@asadiqbal08 -- can you add some context around the catch condition in this code block?